FIAT/kong_mulder_veldhuizen.py

```diff
@@ -77,6 +78,14 @@ def _get_entity_ids(ref_el, degree):
                 1: dict((i, etop[i]) for i in range(3)),
                 2: {0: [i for i in range(15, 30)]},
             }
+    elif degree == 6:
+        if sd == 2:
+            etop = [[6, 12, 3, 13, 7], [9, 15, 4, 14, 8], [10, 16, 5, 17, 11]]
+            entity_ids = {
+                0: dict((i, [i]) for i in range(3)),
+                1: dict((i, etop[i]) for i in range(3)),
+                2: {0: [i for i in range(18, 39)]},
+            }
```

Comment:
Can we improve the ordering here? Say we always want to order vertex dofs first, followed by all dofs of edge 0, edge1, edge2, and finally the interior dofs.